docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,75 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on our high performance computing (HPC) cluster - Discovery, that allows the research community to use private resources on the HPC cluster when they are idle. 
+This new partition has hardware that is not otherwise available to the general research community and, in time, could double the resources available to NURC users. 
+This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
```

Comment:
This paragraph might flow better if we move the third sentence before the second.

Reply:
Done